rmb-sdk-go/peer/peer.go

```diff
@@ -80,14 +81,28 @@ func getIdentity(keytype string, mnemonics string) (substrate.Identity, error) {
 // Call() will panic if called while the directClient's context is canceled.
 func NewPeer(
 	ctx context.Context,
-	keytype string,
 	mnemonics string,
-	relayURL string,
-	session string,
 	sub *substrate.Substrate,
-	enableEncryption bool,
-	handler Handler) (*Peer, error) {
-	identity, err := getIdentity(keytype, mnemonics)
+	handler Handler,
+	opts ...PeerOpt) (*Peer, error) {
+
+	cfg := &peerCfg{
+		relayURL:         "wss://relay.grid.tf",
+		session:          "",
+		enableEncryption: false,
```

Comment:
encryption should be `true` by default actually. only disable explicetly via the option